audiorecorder.1

```diff
@@ -1,21 +1,21 @@
-.TH audiorecorder 1 "github.com/amiaopensource/audiorecorder" "2017\-06\-03" "AMIA Open Source"
+.TH audiorecorder 1 "github.com/amiaopensource/audiorecorder" "2017\-10\-18" "AMIA Open Source"
 .SH NAME
 audiorecorder - Tool for calibration and recording of analog audio sources
 .SH SYNOPSIS
 .B audiorecorder [-e | -p | -m | -b | -h]
 .SH OPTIONS
 .TP
 .B -e
-edit
+edit config
 .TP
 .B -p
-passthrough
+passthrough mode
 .TP
 .B -m
-metadata
+edit metadata for BWF insertion
 .TP
 .B -b
-bext
+embed in the BEXT chunk
```

Comment:
Similar to above, maybe 'embed BEXT chunk (BWF metadata) in target file' or something along those lines?